setup.py

```diff
@@ -25,9 +25,9 @@ def get_version(rel_path):
 setup(name='fairworkflows',
       version=get_version('fairworkflows/_version.py'),
       description='FAIRWorkflows python library',
-      author='Robin Richardson, Djura Smits',
+      author='Robin Richardson, Djura Smits, Sven van den Burg',
       author_email='r.richardson@esciencecenter.nl',
-      url='https://example.org',
+      url='https://github.com/fair-workflows/FAIRWorkbench/',
```

Comment:
This is incorrect now right? It should be fairworkflows

Reply:
Indeed, thanks for spotting!